package.json

```diff
@@ -13,7 +13,8 @@
     "@types/react-flatpickr": "^3.8.4",
     "lerna": "^3.16.4",
     "npm-run-all": "^4.1.5",
-    "snyk": "^1.298.1"
+    "snyk": "^1.298.1",
+    "uniqid": "^5.3.0"
```

Comment:
@lebaphi - shouldn't `snyk` be added to the `package.json` for `packages\client` ?

Reply:
hi @IanMayo, I don't add `snyk `, just add `uniqid ` :D

Reply:
It's ok. I'm pretty sure I can remove it, since the dependency is already in the `client` package.